src/main/kotlin/org/koitharu/kotatsu/parsers/site/HoneyMangaParser.kt

```diff
@@ -0,0 +1,210 @@
+package org.koitharu.kotatsu.core.parser
+
+import okhttp3.Headers
+import okhttp3.Interceptor
+import okhttp3.Response
+import org.json.JSONArray
+import org.json.JSONObject
+import org.koitharu.kotatsu.core.network.CommonHeaders
+import org.koitharu.kotatsu.parsers.MangaLoaderContext
+import org.koitharu.kotatsu.parsers.MangaSourceParser
+import org.koitharu.kotatsu.parsers.PagedMangaParser
+import org.koitharu.kotatsu.parsers.config.ConfigKey
+import org.koitharu.kotatsu.parsers.model.Manga
+import org.koitharu.kotatsu.parsers.model.MangaChapter
+import org.koitharu.kotatsu.parsers.model.MangaPage
+import org.koitharu.kotatsu.parsers.model.MangaSource
+import org.koitharu.kotatsu.parsers.model.MangaState
+import org.koitharu.kotatsu.parsers.model.MangaTag
+import org.koitharu.kotatsu.parsers.model.RATING_UNKNOWN
+import org.koitharu.kotatsu.parsers.model.SortOrder
+import org.koitharu.kotatsu.parsers.util.domain
+import org.koitharu.kotatsu.parsers.util.generateUid
+import org.koitharu.kotatsu.parsers.util.json.getStringOrNull
+import org.koitharu.kotatsu.parsers.util.json.mapJSON
+import org.koitharu.kotatsu.parsers.util.json.mapJSONIndexed
+import org.koitharu.kotatsu.parsers.util.parseJson
+import org.koitharu.kotatsu.parsers.util.parseJsonArray
+import org.koitharu.kotatsu.parsers.util.removeSuffix
+import org.koitharu.kotatsu.parsers.util.tryParse
+import java.text.SimpleDateFormat
+import java.util.EnumSet
+import java.util.Locale
+
+/**
+ * This parser is just for parser development, it should not be used in releases
+ */
+
+private const val PAGE_SIZE = 20
+
+@MangaSourceParser("HONEY-MANGA", "Honey Manga", "uk")
+class HoneyMangaParser(context: MangaLoaderContext) : PagedMangaParser(context, MangaSource.DUMMY, PAGE_SIZE), Interceptor {
+
+	private val urlApi = "https://data.api.$domain"
```

Comment:
Theoretically `$domain` may changes without reinstantiating parser class, so convert this (and fiels below) to getters

Reply:
Done